resources/settings-azure.xml

```diff
@@ -32,14 +32,5 @@
                 </pluginRepository>
             </pluginRepositories>
         </profile>
-        <profile>
```

Comment:
Note that this whole file is misleadingly named and no longer necessary as of #135, but I guess it is here for compatibility with old `Jenkinsfile`s.